cypress/integration/common/beforeEach.js

```diff
@@ -9,12 +9,15 @@ beforeEach(() => {
     // to development defaults does not break a bunch of texts.
     win.INT_TEST_APP_PARAMS = { 
       audience: 'Patient',
-      basePath: '/',
```

Comment:
Was `basePath` removed on purpose for `baseHost`?

Reply:
Copy-paste error! Updating to include baseHost and basePath.

Reply:
👍